ocaml/idl/datamodel.ml

```diff
@@ -5097,6 +5102,7 @@ let network_introduce_params first_rel =
     {param_type=Int; param_name="MTU"; param_doc=""; param_release=first_rel; param_default=None};
     {param_type=Map(String,String); param_name="other_config"; param_doc=""; param_release=first_rel; param_default=None};
     {param_type=String; param_name="bridge"; param_doc=""; param_release=first_rel; param_default=None};
+    {param_type=Bool; param_name="managed"; param_doc=""; param_release=first_rel; param_default=None};
```

Comment:
Yes, only the new param must have `falcon_release`. If it's not defined yet, then please add an entry for it near the top of the file.